server/topic.go

```diff
@@ -814,6 +814,10 @@ func (t *Topic) sendImmediateSubNotifications(asUid types.Uid, acs *MsgAccessMod
 				usersPush(pushRcpt)
 			}
 		}
+	} else if t.cat == types.TopicCatGrp {
+		if pushRcpt := t.pushForGroupSub(asUid, types.TimeNow()); pushRcpt != nil {
```

Comment:
Please also check if `sreg.pkt.Sub.Newsub` is `true`. Otherwise pushes will be sent on all permission changes.

Reply:
Done.